amp.php

```diff
@@ -649,6 +648,8 @@ function amp_prepare_render() {
  * @deprecated This function is not used when 'amp' theme support is added.
  */
 function amp_render() {
+	_deprecated_function( __FUNCTION__, '1.5' );
```

Comment:
One relevant usage: https://wpdirectory.net/search/01DT14Z9DAB3X0VQWZGJPG7XA3

It's in Facebook Instant Articles & Google AMP Pages by PageFrog. However, this plugin has been closed: https://wordpress.org/plugins/pagefrog/